modules/costs/app/helpers/costlog_helper.rb

```diff
@@ -40,6 +40,7 @@ def cost_types_collection_for_select_options(selected_type = nil)
   def user_collection_for_select_options(_options = {})
     Principal
       .possible_assignee(@project)
+      .where(type: 'User')
```

Comment:
Even simpler, this could be written as 

```
User
  .possible_assignee(@project)
  .map { |t| [t.name, t.id] }
```

Reply:
I looked for a similar scope in user as I feared the type would not be correctly applied but looks like it works well. Thanks!